docs/welcome/releases.md

```diff
@@ -21,13 +21,13 @@ Kubernetes version.
 
 ## Compatibility Matrix
 
-| kops version  | k8s 1.11.x | k8s 1.12.x | k8s 1.13.x | k8s 1.14.x | k8s 1.15.x |
+| kops version  | k8s 1.12.x | k8s 1.13.x | k8s 1.14.x | k8s 1.15.x | k8s 1.16.x |
```

Comment:
I feel like the information in this chart is the biggest stumbling block for most people.  That said, the common failure mode is running a newer version of k8s than kops, and I believe we have an error message for that now, and we are hopefully about to ~ catch up, so may not be as big of a concern any more?

Reply:
I think this is less of a concern now that we have that warning.

Reply:
Agreed. The chart was added to help people but often confused them even more. Regardless, I feel like lack of clarity around supported versions was the real issue (which we've addresses very heavily lately).  I think we should leave this for now and come back another time if it should be removed from the releases page.